### ISSUE TITLE
Web demo is not launching any results. Might be disconnected from the model.

### ISSUE THREAD

**Issue by Gertie01** [kingoflolz/mesh-transformer-jax#241]

**Comment by simon-clematide** [comment]
Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)

**Comment by Gertie01** [comment]
How about an update?

**Comment by Gertie01** [comment]
> How about an update? No free trials. No subscriptions. Keep it unlimited.

Do you reply?

**Comment by Gertie01** [comment]
Try making a new web demo.

> Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)

Fix the web demo.

**Comment by Gertie01** [comment]
Fix the web demo.

**Comment by atrdev-rgb** [comment]
Unable to connect to the model. Please try again.

EleutherAI 2021

**Comment by Gertie01** [comment]
> Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)

Try fixing the web demo.

**Comment by Gertie01** [comment]
> Unable to connect to the model. Please try again.
> 
> EleutherAI 2021

Not helping.

**Comment by Gertie01** [comment]
> Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)

Plan A: Try making a new web demo.
Or
Plan B: Try fixing the web demo.

What's your choice?

**Comment by simon-clematide** [comment]
I don't know, would a gradio app be possible?

**Comment by Gertie01** [comment]
Try Plan B first.

**Comment by Gertie01** [comment]
Any luck?

**Comment by Gertie01** [comment]
> I don't know, would a gradio app be possible?

Yes.

**Comment by Gertie01** [comment]
> I don't know, would a gradio app be possible?

Yes.

**Comment by Gertie01** [comment]
> I don't know, would a gradio app be possible?

I say "Yes."

**Comment by Gertie01** [comment]
> I don't know, would a gradio app be possible?

Good idea!

**Comment by Gertie01** [comment]
> > I don't know, would a gradio app be possible?
> 
> I say "Yes."

Do you respond?

**Comment by Gertie01** [comment]
> > > I don't know, would a gradio app be possible?
> > 
> > 
> > I say "Yes."
> 
> Do you respond?

I just want you to replace the old web demo with a new demo.

**Comment by simon-clematide** [comment]
Ok, sorry, I didn't understand the intention of your messages. So, in short, you suggest that I just create a new web demo. I can give it a try.

**Comment by Gertie01** [comment]
> Ok, sorry, I didn't understand the intention of your messages. So, in short, you suggest that I just create a new web demo. I can give it a try.

Is it gonna be 100% FREE and 100% UNLIMITED?

**Comment by Gertie01** [comment]
> > Ok, sorry, I didn't understand the intention of your messages. So, in short, you suggest that I just create a new web demo. I can give it a try.
> 
> Is it gonna be 100% FREE and 100% UNLIMITED?

It’s a suggestion.

**Comment by Gertie01** [comment]
How’s the new web demo going?

**Comment by Gertie01** [comment]
> Ok, sorry, I didn't understand the intention of your messages. So, in short, you suggest that I just create a new web demo. I can give it a try.

Try putting the model back to Mystic.ai.

**Comment by Gertie01** [comment]
6b.eleuther.ai mystic model is down for GPT-J-6B.

**Comment by Gertie01** [comment]
> 6b.eleuther.ai mystic model is down for GPT-J-6B.

I just want it back up.

**Comment by Gertie01** [comment]
> > 6b.eleuther.ai mystic model is down for GPT-J-6B.
> 
> I just want it back up.

Do you respond?

**Comment by Gertie01** [comment]
> > Ok, sorry, I didn't understand the intention of your messages. So, in short, you suggest that I just create a new web demo. I can give it a try.
> 
> Try putting the model back to Mystic.ai.



> Ok, sorry, I didn't understand the intention of your messages. So, in short, you suggest that I just create a new web demo. I can give it a try.

6b.eleuther.ai mystic model is down for GPT-J-6B.

**Comment by Gertie01** [comment]
> Ok, sorry, I didn't understand the intention of your messages. So, in short, you suggest that I just create a new web demo. I can give it a try.

6b.eleuther.ai mystic model is down for GPT-J-6B.

**Comment by Gertie01** [comment]
> Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)

Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.

**Comment by Gertie01** [comment]
> > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> 
> Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.

Do you respond?

**Comment by Gertie01** [comment]
> > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > 
> > 
> > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> 
> Do you respond?

Do you respond?

**Comment by Gertie01** [comment]
> > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > 
> > > 
> > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > 
> > 
> > Do you respond?
> 
> Do you respond?

Respond now.

**Comment by Gertie01** [comment]
> > > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > > 
> > > > 
> > > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > > 
> > > 
> > > Do you respond?
> > 
> > 
> > Do you respond?
> 
> Respond now.

Mom: Colin. Do you want me to take you out of Deviantart?

**Comment by Gertie01** [comment]
> > > > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > > > 
> > > > > 
> > > > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > > > 
> > > > 
> > > > Do you respond?
> > > 
> > > 
> > > Do you respond?
> > 
> > 
> > Respond now.
> 
> Mom: Colin. Do you want me to take you out of Deviantart?

Mom: Answer me.

**Comment by Gertie01** [comment]
> > > > > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > > > > 
> > > > > > 
> > > > > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > > > > 
> > > > > 
> > > > > Do you respond?
> > > > 
> > > > 
> > > > Do you respond?
> > > 
> > > 
> > > Respond now.
> > 
> > 
> > Mom: Colin. Do you want me to take you out of Deviantart?
> 
> Mom: Answer me.

simon-clematide responded.

**Comment by Gertie01** [comment]
> > > > > > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > > > > > 
> > > > > > > 
> > > > > > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > > > > > 
> > > > > > 
> > > > > > Do you respond?
> > > > > 
> > > > > 
> > > > > Do you respond?
> > > > 
> > > > 
> > > > Respond now.
> > > 
> > > 
> > > Mom: Colin. Do you want me to take you out of Deviantart?
> > 
> > 
> > Mom: Answer me.
> 
> simon-clematide responded.

Mom: No. Answer me. Do you want me to take you out of Deviantart?

**Comment by Gertie01** [comment]
> > > > > > > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > > > > > > 
> > > > > > > 
> > > > > > > Do you respond?
> > > > > > 
> > > > > > 
> > > > > > Do you respond?
> > > > > 
> > > > > 
> > > > > Respond now.
> > > > 
> > > > 
> > > > Mom: Colin. Do you want me to take you out of Deviantart?
> > > 
> > > 
> > > Mom: Answer me.
> > 
> > 
> > simon-clematide responded.
> 
> Mom: No. Answer me. Do you want me to take you out of Deviantart?

NO! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.

**Comment by Gertie01** [comment]
> > > > > > > > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Do you respond?
> > > > > > > 
> > > > > > > 
> > > > > > > Do you respond?
> > > > > > 
> > > > > > 
> > > > > > Respond now.
> > > > > 
> > > > > 
> > > > > Mom: Colin. Do you want me to take you out of Deviantart?
> > > > 
> > > > 
> > > > Mom: Answer me.
> > > 
> > > 
> > > simon-clematide responded.
> > 
> > 
> > Mom: No. Answer me. Do you want me to take you out of Deviantart?
> 
> NO! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.

Mom: No. I sent simon-clematide home forever. Simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?

**Comment by Gertie01** [comment]
> > NO! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.

NO! NOT HOME! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.

**Comment by Gertie01** [comment]
> > > > > > > > > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Do you respond?
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Do you respond?
> > > > > > > 
> > > > > > > 
> > > > > > > Respond now.
> > > > > > 
> > > > > > 
> > > > > > Mom: Colin. Do you want me to take you out of Deviantart?
> > > > > 
> > > > > 
> > > > > Mom: Answer me.
> > > > 
> > > > 
> > > > simon-clematide responded.
> > > 
> > > 
> > > Mom: No. Answer me. Do you want me to take you out of Deviantart?
> > 
> > 
> > NO! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.
> 
> Mom: No. I sent simon-clematide home forever. Simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?

NO! NOT HOME! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.

**Comment by Gertie01** [comment]
> > > > > > > > > > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > > > > > > > > > 
> > > > > > > > > > > 
> > > > > > > > > > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > Do you respond?
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Do you respond?
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Respond now.
> > > > > > > 
> > > > > > > 
> > > > > > > Mom: Colin. Do you want me to take you out of Deviantart?
> > > > > > 
> > > > > > 
> > > > > > Mom: Answer me.
> > > > > 
> > > > > 
> > > > > simon-clematide responded.
> > > > 
> > > > 
> > > > Mom: No. Answer me. Do you want me to take you out of Deviantart?
> > > 
> > > 
> > > NO! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.
> > 
> > 
> > Mom: No. I sent simon-clematide home forever. Simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?
> 
> NO! NOT HOME! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.

Mom: No. simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?

**Comment by Gertie01** [comment]
> > > > > > > > > > > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > > > > > > > > > > 
> > > > > > > > > > > > 
> > > > > > > > > > > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > > > > > > > > > > 
> > > > > > > > > > > 
> > > > > > > > > > > Do you respond?
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > Do you respond?
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Respond now.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Mom: Colin. Do you want me to take you out of Deviantart?
> > > > > > > 
> > > > > > > 
> > > > > > > Mom: Answer me.
> > > > > > 
> > > > > > 
> > > > > > simon-clematide responded.
> > > > > 
> > > > > 
> > > > > Mom: No. Answer me. Do you want me to take you out of Deviantart?
> > > > 
> > > > 
> > > > NO! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.
> > > 
> > > 
> > > Mom: No. I sent simon-clematide home forever. Simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?
> > 
> > 
> > NO! NOT HOME! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.
> 
> Mom: No. simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?

NO! NOT HOME! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.

**Comment by Gertie01** [comment]
> > > > > > > > > > > > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > > > > > > > > > > > 
> > > > > > > > > > > > > 
> > > > > > > > > > > > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > > > > > > > > > > > 
> > > > > > > > > > > > 
> > > > > > > > > > > > Do you respond?
> > > > > > > > > > > 
> > > > > > > > > > > 
> > > > > > > > > > > Do you respond?
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > Respond now.
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Mom: Colin. Do you want me to take you out of Deviantart?
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Mom: Answer me.
> > > > > > > 
> > > > > > > 
> > > > > > > simon-clematide responded.
> > > > > > 
> > > > > > 
> > > > > > Mom: No. Answer me. Do you want me to take you out of Deviantart?
> > > > > 
> > > > > 
> > > > > NO! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.
> > > > 
> > > > 
> > > > Mom: No. I sent simon-clematide home forever. Simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?
> > > 
> > > 
> > > NO! NOT HOME! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.
> > 
> > 
> > Mom: No. simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?
> 
> NO! NOT HOME! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.

Mom: No. simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?

**Comment by Gertie01** [comment]
> > Unable to connect to the model. Please try again.
> > EleutherAI 2021
> 
> Not helping.

Sorry about that. Proceed.

**Comment by Gertie01** [comment]
> > > > > > > > > > > > > > > Are there any plans to revive the demo? It was actually very nice to have such a demo for instruction purposes:-)
> > > > > > > > > > > > > > 
> > > > > > > > > > > > > > 
> > > > > > > > > > > > > > Ask Mystic.ai for a model request via Discord or upload the model to Mystic.ai.
> > > > > > > > > > > > > 
> > > > > > > > > > > > > 
> > > > > > > > > > > > > Do you respond?
> > > > > > > > > > > > 
> > > > > > > > > > > > 
> > > > > > > > > > > > Do you respond?
> > > > > > > > > > > 
> > > > > > > > > > > 
> > > > > > > > > > > Respond now.
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > Mom: Colin. Do you want me to take you out of Deviantart?
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Mom: Answer me.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > simon-clematide responded.
> > > > > > > 
> > > > > > > 
> > > > > > > Mom: No. Answer me. Do you want me to take you out of Deviantart?
> > > > > > 
> > > > > > 
> > > > > > NO! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.
> > > > > 
> > > > > 
> > > > > Mom: No. I sent simon-clematide home forever. Simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?
> > > > 
> > > > 
> > > > NO! NOT HOME! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.
> > > 
> > > 
> > > Mom: No. simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?
> > 
> > 
> > NO! NOT HOME! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS.
> 
> Mom: No. simon-clematide can’t respond. Answer me do you want me to take you out of Deviantart. Do you want me to take you out of Deviantart?

NO! NOT HOME! I CAN’T ANSWER YOU DO I WANT YOU TO TAKE ME OUT OF DEVIANTART! SIMON-CLEMATIDE RESPONDS! SIMON-CLEMATIDE, RESPOND! We just want 6b.eleuther.ai mystic model back up.

**Comment by ItzDFPlayer** [comment]
wtf is going on here

**Comment by Gertie01** [comment]
> wtf is going on here

simon-clematide can’t respond. He's sent home by Mom. Mom is demanding me to answer her do I want her to sign me out of Deviantart so that Me99999 and sophiestrawberry can take me back to their school on Discord and resume forcing me to say something else forever.
